### PR TITLE
Update to use embark 6.0.0

### PR DESCRIPTION
**Pull request by 3esmit**

### CODE REVIEW

File: contracts/common/MessageSigned.sol [hunk 1]
```diff
@@ -70,8 +69,10 @@ contract MessageSigned {
             // use the second best option, 'and'
             v := and(mload(add(_signature, 65)), 0xff)
         }
-
-        require(v == 27 || v == 28);
+        if (v < 27) {
+            v += 27;
```
Comment (0kok0) [inline contracts/common/MessageSigned.sol:73]: Why do we allow `v = 0, 1` as well?

File: contracts/ens/PublicResolver.sol [hunk 2]
```diff
@@ -10,20 +10,20 @@ contract PublicResolver {
...
     bytes4 constant INTERFACE_META_ID = 0x01ffc9a7;
     bytes4 constant ADDR_INTERFACE_ID = 0x3b3b57de;
-    bytes4 constant CONTENT_INTERFACE_ID = 0xd8389dc5;
     bytes4 constant NAME_INTERFACE_ID = 0x691f3431;
     bytes4 constant ABI_INTERFACE_ID = 0x2203ab56;
     bytes4 constant PUBKEY_INTERFACE_ID = 0xc8690233;
     bytes4 constant TEXT_INTERFACE_ID = 0x59d1d43c;
-    bytes4 constant MULTIHASH_INTERFACE_ID = 0xe89401a1;
+    bytes4 constant CONTENTHASH_INTERFACE_ID = 0xbc1c58d1;
+    bytes4 constant INTERFACE_INTERFACE_ID = bytes4(keccak256("interfaceImplementer(bytes32,bytes4)"));
```
Comment (0kok0) [inline contracts/ens/PublicResolver.sol:18]: Is there any trouble if there's a collision for this id?

**Comment by rajeevgopalakrishna** [comment]
Recording our decision here - as discussed, this PR only changes Embark dependencies. So the Crytic issues reported (all on legacy contracts) for this PR will be carried over for reviewing/resolution to the other PRs after rebasing them to this PR merge.

**Comment by lokera07** [comment]
> _No se proporcionó descripción._



> _No se proporcionó descripción._

Ya quedo

**Pull request merged**